lang/src/org/partiql/lang/ast/passes/AstRewriterBase.kt

```diff
@@ -31,8 +32,9 @@ interface AstRewriter {
 @Deprecated("New rewriters should implement PIG's PartiqlAst.VisitorTransform instead")
```

Comment:
Could update this comment to use `VisitorTransformBase` rather than `PartiqlAst.VisitorTransform`.

Reply:
indeed. next rev.